src/sushi.sol

```diff
@@ -110,14 +122,10 @@ contract SushiJoin is CropJoin {
     }
 
     // --- Administration ---
-    function file(bytes32 what, uint256 data) external auth {
-        if (what == "pid") pid = data;
-        else revert("SushiJoin/file-unrecognized-param");
-        emit File(what, data);
-    }
     function file(bytes32 what, address data) external auth {
-        if (what == "initialMigrator") initialMigrator = data;
-        else if (what == "timelockOwner") timelockOwner = TimelockLike(data);
+        if (what == "migrator") migrator = data;
+        else if (what == "rewarder") rewarder = data;
+        else if (what == "timelock") timelock = TimelockLike(data);
```

Comment:
Not sure how everyone feels about mixing `address` and interface values like this. I'd rather see these things defined as `address` and then invoked as `Interface(addr)` throughout, but I think we're inconsistent on it.

Reply:
My intuition is that we use `address` when there is no interface dependency, and we use an interface when we do. In this contract we don't really care about the interface for `migrator` and `rewarder`, we only care about what address they are set to and if that changes. I believe this is consistent with what I've seen elsewhere in dss.

Reply:
For me it's largely a judgment call based on readability and convenience. If something is always used as an interface, sometimes it makes sense to type it as such to cut down on the casting.